assets/blocks/quiz/question-block/question.scss

```diff
@@ -120,18 +120,23 @@ $question-input-border-color: var(--wp--preset--color--foreground, #155E65 );
 .sensei-lms-question__answer-feedback {
 	--color: #DCDCDE;
 	$horizontal-padding: 24px;
-	border: 1px solid var(--color);
+	border: 2px solid var(--border-color);
 	border-radius: 1px;
+	font-family: var(--wp--preset--font-family--system);
```

Comment:
This CSS variable is only available for block themes, and only those that explicitly have a custom font named `system`? We should explicitly fall back to `--wp--preset--font-family--body-font`, which it's currently defaulting to anyway.

Reply:
I've added the explicit fallback here https://github.com/Automattic/sensei/pull/7171/commits/272a9e18549e9cabe52d9256d2744be670f77804. Thanks!